style.css

```diff
@@ -16267,3 +16267,44 @@ main.main-sidebar {
   opacity: 0.7;
   border-radius: 50%;
 }
+
+/* Header Accordian / Collapsible Headers */
+.accordion {
+  background-color: #a5aeb5;
```

Comment:
I think it's better practice to make variables of these colors so you can use them again next time. You should be able to find examples of them in `_custom.scss` file.